src/views/auth/error.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'default.html' %}
+{% set title = 'Authentication Error' %}
+{% block pageHeading %}Oops!{% endblock %}
```

Comment:
Will just change it to "We're Sorry" for everything and if we need to change it for other errors we'll look at it again